draft-ietf-mls-protocol.md

```diff
@@ -1659,11 +1659,16 @@ group and epoch.
 
 ~~~~~
 struct {
-    GroupContext context;
-
-    opaque group_id<0..255>;
-    uint64 epoch;
     Sender sender;
+    select (MLSPlaintextTBS.sender.sender_type) {
+        case member:
+            GroupContext context;
+
+        case preconfigured, new_member:
+            opaque group_id<0..255>;
+            uint64 epoch;
+    }
+
```

Comment:
- Also included extensions
- I think this struct is technically un-parse-able because the select is before the field that the select is on, but implementations can probably cheat to get it to work

Reply:
It doesn't actually need to be parseable, just serializable.  If you wanted to make it parseable, you could change the `select` to an `optional<GroupContext>` and mandate is presence when `sender_type == member`.